apps/src/templates/projects/PersonalProjectsTableActionsCell.jsx

```diff
@@ -21,6 +21,9 @@ export const styles = {
   xIcon: {
     paddingRight: 5,
   },
+  deleteOption: {
```

Comment:
Could you migrate the inline styles in this file to an scss module? I believe we have an in progress initiative to do this as we touch stuff - [Dev Guide - in progress initiatives](https://docs.google.com/document/d/1Yxek5dqYKLJUraB17yCdEFLlJvlOiB1zhSWZAuu3zPU/edit#bookmark=id.gu8vyypw2a5e)

Reply:
Sure. This is a little tricker than I initially thought so I logged as a fast follow-up ticket at https://codedotorg.atlassian.net/browse/LABS-544. Thanks!

Reply:
Updated! A couple updates pushed since you last approved. Thanks for all your help!